webpack.backend.config.babel.js

```diff
@@ -0,0 +1,34 @@
+import webpack from 'webpack';
```

Comment:
This is the `webpack` configuration file for our backend code. (we will also have a `webpack` configuration file for our frontend code soon)

Probably in future work this is actually going to become the `api` `webpack` configuration file. Because we're going to have a backend AND an api.